flake.nix

```
@@ -249,6 +270,29 @@
             mkdir $out # success
           '';
 
+          checks.age-plugin = pkgs.runCommand "age-plugin" { } ''
```

The test also requires `recursive-nix`: 

```suggestion
          checks.age-plugin = pkgs.runCommand "age-plugin"
            {
              buildInputs = [ exampleRagePlugin ];
              requiredSystemFeatures = [ "recursive-nix" ];
            }
            ''
```

---

Yeah just saw it, pushed already 🙈